Python-Home-Challenges/recurring word.py

```diff
@@ -0,0 +1,22 @@
+
+where_file = input('Enter the file route: ')
+my_file = open(where_file,"r")
+print("Your file text is:")
+for x in my_file:
+  print(x)
+print("---------------")
+words_in_text = dict()
+my_file = open(where_file,"r")
```

Comment:
you dont need to open file again, also, file must be closed after ward. best practice is to use "open...with" when working with files.

Reply:
roger that!
will be fixed.